gamemode/modules/base/sv_interface.lua

```diff
@@ -67,6 +67,12 @@ DarkRP.retrievePlayerData = DarkRP.stub{
             description = "The function that receives the information.",
             type = "function",
             optional = false
+        },
+        {
+            name = "failure",
+            description = "The function that is called when the information cannot be retrieved.",
+            type = "function",
+            optional = false
```

Comment:
let's call it a microptimisation then :v

we won't have to poll the database because we've already checked whether it's a SteamID or not